packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/rebac/askem/RebacUser.java

```diff
@@ -13,26 +13,35 @@ public class RebacUser extends RebacObject {
 	private final ReBACService reBACService;
 
 	private final boolean serviceUser;
+	private final boolean adminServiceUser;
 
 	public RebacUser(final String id, final ReBACService reBACService) {
 		super(id);
 		this.reBACService = reBACService;
 
 		serviceUser = ReBACService.isServiceUser(id);
+		adminServiceUser = ReBACService.isAdminServiceUser(id);
 	}
 
 	@Override
 	public SchemaObject getSchemaObject() {
 		return new SchemaObject(Schema.Type.USER, getId());
 	}
 
+	public boolean isAdmin() {
+		return adminServiceUser;
+	}
+
 	public boolean can(final RebacObject rebacObject, final Schema.Permission permission) throws Exception {
-		if (serviceUser) return true;
+		if (serviceUser || adminServiceUser) return true;
 		if (rebacObject.getId().isEmpty()) return false;
 		return reBACService.can(getSchemaObject(), permission, rebacObject.getSchemaObject());
 	}
 
 	public boolean isMemberOf(final RebacGroup rebacGroup) throws Exception {
+		if (adminServiceUser) {
```

Comment:
You should use `isAdmin` if we start to add other admins by hand like this. Would make cleaning up/refactoring easier.

Reply:
Fixed